### PR TITLE
Allow rails 7 gems in gemspec

### PR DESCRIPTION
**Pull request by jrafanie**
Allow rails 7 gems in gemspec

### CODE REVIEW

**Comment by jrafanie** [comment]
Note, we only use activesupport for Time.current and symbolize_keys.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -12,13 +12,12 @@ jobs:
     strategy:
       matrix:
         ruby-version:
-        - '2.5'
-        - '2.6'
         - '2.7'
         - '3.0'
```
Comment (bdunne) [inline .github/workflows/ci.yaml:16]: Should this also add ruby 3.1?

Comment (jrafanie) [reply .github/workflows/ci.yaml:16]: yeah, I was debating it in all my prs and didn't in some.  I can add it.  I should probably circle around and do it for all of them that are standalone.

Comment (jrafanie) [reply .github/workflows/ci.yaml:16]: good call, tests are broken on ruby 3.1

Comment (jrafanie) [reply .github/workflows/ci.yaml:16]: fixed

**Pull request merged**